lib/prerender.js

```diff
@@ -92,7 +98,10 @@ class Prerender extends Plugin {
       request: {
         url,
         headers: {
-          host
+          host,
+          'x-broccoli': {
```

Comment:
Do we want the rest of FastBoot's x-broccoli?

- Referenced by middleware: https://github.com/ember-fastboot/ember-cli-fastboot/blob/c83e885fb51c5b01f39e966ee54fb865d5bf41ca/index.js#L307
- Otherwise not sure where this is defined

Reply:
https://github.com/ember-cli/broccoli-middleware/blob/31de4a8a3808cd91e0cffe9c017d1c65c8303963/lib/watcher-middleware.js#L20-L30

I don't know  how much this matters, I'm fine with only including things that are critical to make it work. It seems bad if fastboot apps won't run correctly without this header.

Reply:
Aha! Nice find. Had you used that before?

I wouldn't consider this implementation "critical to make it work"; there are other approaches we can take (eg allow plugins / users to expose their own metadata, some of which may be filled in at runtime). I do think this is the simplest.

Re use case, I am loading JSON + MD files from the compiled output as in-app data. When building, the app does not have access to a local server, so can't make the HTTP requests. Passing the build dir here matches how served FastBoot expresses its path, and allows me to read files from built source.

Sample source: https://github.com/anulman/ember-cli-hyde/blob/3975d90c02c1ab1d4e2e1d03c524736617d6ca3f/addon/services/hyde.js#L15-L34